### ISSUE TITLE
similar function like "inconsistent_enumerated_columns" but then for column labels

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#1856]
### Standard

CDISC SEND

### Reference Rule ID(s)

SEND313

### JIRA Ticket

https://jira.cdisc.org/browse/CORERULES-746

### Proposed Rule YAML

```YAML
N/A
```

### Attach any sample test data file(s)

For SEND313 "When using COVAL1-COVALn, the **variable labe**l should match the corresponding variable name (e.g., Comment1, Comment2)."
E.g. when we have COVAL1 as a variable, its label is expected to be "Comment1" or "Comment 1".
For COVAL2, the label is expected to be "Comment2" or "Comment 2".
We do already have the function "inconsistent_enumerated_columns", but we need something similar for the labels. 
The function could e.g. be named "inconsistent_enumerated_column_labels".

### Expected output

boolean whether the label of COVALn has the correct label
